### PR TITLE
Ron A. has updated 1 entry in the bibliography

### PR DESCRIPTION
**Pull request by RonVin-AspIII**

### CODE REVIEW

File: data/bibliography.json [hunk 1]
```diff
@@ -6534,7 +6534,8 @@
     {
       "id": "9296070/FFAKSC38",
       "type": "report",
-      "title": "A Display Oriented Programmers Assistant"
+      "title": "A display Oriented Programmers Assistant",
```
Comment (nbriggs) [inline data/bibliography.json:6537]: "A Display" here also?

**Review by nbriggs — COMMENTED** [review]
It would help with the review if you'd add a note about the intent of the change.

Fix grouping? fix sort order? remove link to bitsavers? change the title from the name of the PDF to the actual article title?  Something else?

Without that, we're left guessing as to whether what changed was correct for the intended purpose.

**Comment by masinter** [comment]
my bad... I was walking ron through making a change and seeing the result on interlisp.org. but no need to actually do this.

**Comment by RonVin-AspIII** [comment]
Apologies for the confusing and sudden push requests! But now that Larry
has guided me through the pull request process, I will definitely take the
feedback into consideration for future commits. Thank you for your time!

Very Respectfully,
Ron Vincent Aspuria III

**Pull request closed without merging**